lib/DDGC/DB/Result/User/Blog.pm

```diff
@@ -1,202 +1,108 @@
 package DDGC::DB::Result::User::Blog;
+
 # ABSTRACT: Result class of blog posts
 
-use Moose;
-use MooseX::NonMoose;
-extends 'DDGC::DB::Base::Result';
+use Moo;
+extends 'DBIx::Class::Core';
 use DBIx::Class::Candy;
 use DateTime::Format::RSS;
-use namespace::autoclean;
 
 table 'user_blog';
 
-sub u {
-	my ( $self ) = @_;
-	if ($self->company_blog) {
-		return ['Blog', 'post', $self->uri];
-	} else {
-		return ['Userpage::Blog', 'post', $self->user->username, $self->uri];
-	}
-}
-
 column id => {
-	data_type => 'bigint',
-	is_auto_increment => 1,
+    data_type         => 'bigint',
+    is_auto_increment => 1,
 };
 primary_key 'id';
 
 column users_id => {
-	data_type => 'bigint',
-	is_nullable => 0,
+    data_type   => 'bigint',
+    is_nullable => 0,
 };
 
 column translation_of_id => {
-	data_type => 'bigint',
-	is_nullable => 1,
+    data_type   => 'bigint',
+    is_nullable => 1,
 };
 
 column title => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
 
 column uri => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
 
 column teaser => {
-	data_type => 'text',
-	is_nullable => 0,
+    data_type   => 'text',
+    is_nullable => 0,
 };
-sub html_teaser {
```

Comment:
Can't change the names just yet - it's the post fragment that goes in the index.